pyrostest/ros_test.py

```diff
@@ -49,9 +47,8 @@ def __init__(self, topic, msg_type, queue_size):
     def send(self, value):
         """Sends data to be publoshed to the mocked topic.
         """
+        time.sleep(.5)
```

Comment:
Preferably this would go in the init method, but that didn't work, it merits some further investigation perhaps, but yeah its tricky.